include/fuse.h

```diff
@@ -664,6 +667,10 @@ struct fuse_operations {
 	 */
 	int (*bmap) (const char *, size_t blocksize, uint64_t *idx);
 
+#if FUSE_USE_VERSION < 35
```

Comment:
What's the rationale for this change? It doesn't seem related to Doxygen to me...